charts/ceph-csi-cephfs/values.yaml

```diff
@@ -191,6 +191,56 @@ topology:
     - failure-domain/region
     - failure-domain/zone
 
+storageClass:
+  # Specifies whether the Storage class should be created
+  create: true
+  name: csi-cephfs-sc
+  parameters:
+    # String representing a Ceph cluster to provision storage from.
+    # Should be unique across all Ceph clusters in use for provisioning,
+    # cannot be greater than 36 bytes in length, and should remain immutable for
+    # the lifetime of the StorageClass in use.
+    clusterID: <cluster-ID>
+    # (required) CephFS filesystem name into which the volume shall be created
+    # eg: fsName: myfs
+    fsName: <cephfs-name>
+    # (optional) Ceph pool into which volume data shall be stored
+    # pool: <cephfs-data-pool>
+
+    # (optional) Comma separated string of Ceph-fuse mount options.
+    # For eg:
+    # fuseMountOptions: debug
+
+    # (optional) Comma separated string of Cephfs kernel mount options.
+    # Check man mount.ceph for mount options. For eg:
+    # kernelMountOptions: readdir_max_bytes=1048576,norbytes
+
+    # (optional) The driver can use either ceph-fuse (fuse) or
+    # ceph kernelclient (kernel).
+    # If omitted, default volume mounter will be used - this is
+    # determined by probing for ceph-fuse and mount.ceph
+    # mounter: kernel
+
+    # (optional) Prefix to use for naming subvolumes.
+    # If omitted, defaults to "csi-vol-".
+    # volumeNamePrefix: "foo-bar-"
+  reclaimPolicy: Delete
+  allowVolumeExpansion: true
+  mountOptions:
+    - discard
```

Comment:
mountoptions should not be enabled by default

Reply:
Done